data-builder/parse_items.py

```diff
@@ -155,6 +169,22 @@ def get_items_from_page(itemPageURL, sets):
                 item['crafting'].append(affix['name'])
                 remove.append(affix)
 
+            # if enhancement bonus found on item we need to translate that enhancement bonus to something else
+            if affix['name'] == 'Enhancement Bonus':
+
+                # for armor and shield items - enhancement bonus becomes an enhancement type bonus to armor class
+                if ((item['slot'] == 'Armor') or (item['slot'] == 'Offhand')):
```

Comment:
I don't think we can check just the `slot` here, since you can have a weapon in your offhand.